src/pages/library/WorkflowsLibrary.tsx

```diff
@@ -51,11 +53,21 @@ interface WorkflowSourceBoxProps {
   description: string;
   url: string;
   logoFilePath: string;
+  metricsEventName: MetricsEventName;
 }
 
 const WorkflowSourceBox = (props: WorkflowSourceBoxProps) => {
+  const sendMetrics = () => {
+    // don't send metrics if Broad Methods Repo card and links are shown in UI
+    // TODO: remove this if condition when feature flag FIRECLOUD_UI_MIGRATION is removed.
```

Comment:
Can this condition be based on the feature flag itself?

Reply:
I did think of that. But this `WorkflowSourceBox` component is used by 2 cards one of which only uses the feature flag. I also thought of adding another boolean field in the WorkflowSourceBoxProps that indicates whether sending metrics is enabled or not. But I thought using `Broad Methods Repo` is the easiest approach, especially when these conditions need to be removed, since we want to anyways remove the feature flag next sprint.